### ISSUE TITLE
Capacitor V4 compatibility

### ISSUE THREAD

**Issue by de-dan** [martinkasa/capacitor-secure-storage-plugin#58]
Hey, thanks for your lovely work!

Unfortunately this lib is not (yet) compatible with the newest capacitor release:

```
npm ERR! Found: @capacitor/core@4.0.0
npm ERR! node_modules/@capacitor/core
npm ERR!   optional @capacitor/core@"^4.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @capacitor/core@"^3.0.0" from capacitor-secure-storage-plugin@0.7.1
npm ERR! node_modules/capacitor-secure-storage-plugin
npm ERR!   optional capacitor-secure-storage-plugin@"^0.7.1" from the root project
```

**Comment by de-dan** [comment]
Created PR #59

**Comment by martinkasa** [comment]
Fixed by #59
